packages/frontend/src/ee/CommercialRoutes.tsx

```diff
@@ -82,6 +83,14 @@ const COMMERCIAL_AI_AGENTS_ROUTES: RouteObject[] = [
                 index: true,
                 element: <AgentsListPage />,
             },
+            {
+                path: 'new',
+                element: <ProjectAiAgentEditPage isCreateMode />,
+            },
+            {
+                path: ':agentUuid/edit',
+                element: <ProjectAiAgentEditPage />,
+            },
             {
```

Comment:
We need to protect these, so only admins can see these

Reply:
🫡

Reply:
addressed in `ProjectAiAgentEditPage` - removed slack alert for non-admins bc they won't ever see that

added a redirect to main ai-agents page if they don't have enough permissions - wdyt @tatianainama ?